### ISSUE TITLE
Improve the algorithm for finding the next link

### ISSUE THREAD

**Issue by DoomPtrl** [sixcious/infy-scroll#73]
### Feature Request

Can you please make Infy compatible with websites with only pagination links(1, 2, …) with no next button?

### Adherence

- [X] My feature request is an actual feature and is not related to making it work with a specific website

**Comment by sixcious** [comment]
Hi DoomPtrl, thanks for making this feature request!

Yes, this one is on my [Roadmap](https://github.com/sixcious/infy-scroll/wiki/Roadmap). Infy currently only employs a keyword-based algorithm for auto-detecting the next link and click element, so if a site decides to not use any sort of descriptive HTML, it won't find them and you'll need to construct the path manually or by using the Element Picker. I recommend to try using [Pagetual](https://github.com/hoothin/UserScripts/tree/master/Pagetual), which I believe should be able to do this.

Hopefully, this feature can be added into the next major version of Infy (though I'm not sure when that will be!). We can keep this issue open until then.

Thank you! 💜

**Comment by sixcious** [comment]
Hi again DoomPtrl, sorry for the long delay! I added a new and unique Auto Detect Pattern algorithm option in the newest version of Infy Scroll today (Version 8.1). I've spent a *lot* of time working on it, and while it won't work on every site, it should work on a lot more sites in auto detecting the next link. I hope to continually improve it in the future.

Thank you for making this feature request! 💜
